src/get-started/create-service.md

```diff
@@ -351,6 +393,16 @@ struct CryptocurrencyApi {
 }
 ```
 
+It is also necessary to create a structure that will be returned by the REST API:
```

Comment:
Awkward phrasing; propose to change to

> We also define a type for transaction response.

Also, it could be moved to the "API for Transactions" section.